requirements.txt

```diff
@@ -1,4 +1,4 @@
 sycret
-syft @ git+https://github.com/OpenMined/PySyft@dev#egg=syft&subdirectory=packages/syft
+syft @ git+https://github.com/OpenMined/PySyft@bb093294e98aa2be81412318fe8314cf6205c86b#egg=syft&subdirectory=packages/syft
```

Comment:
Q: Can't we put dev? (or dev i still not working)

Reply:
I think it is  fixed , I will revert back :+1: